README.md

```diff
@@ -18,16 +18,22 @@ Sockeye requires:
 
 ## Installation
 
-### For AWS DeepLearning AMI users
+There are several options for installing Sockeye and it's dependencies. Below we list several alterantives and the
+corresponding instructions.
 
-AWS DeepLearning AMI users only need to run the following line to install sockeye:
+### Either: AWS DeepLearning AMI
```

Comment:
I would remove the "Either" and "Or" below

Reply:
I just thought that it wasn't entirely clear there these were different options and I wanted to make this as clear as possible.